src/Microsoft.Docs.Template/Models/TableOfContents.cs

```diff
@@ -7,17 +7,27 @@
 
 namespace Microsoft.Docs.Build
 {
-    internal class TableOfContentsItem
+    public class TableOfContentsModel
+    {
+        [JsonProperty(PropertyName = "metadata")]
+        public JObject Metadata { get; set; }
+
+        [JsonProperty(PropertyName = "items")]
+        public List<TableOfContentsItem> Items { get; set; }
+    }
+
+    public class TableOfContentsItem
```

Comment:
>public class [](start = 4, length = 12)

Does StyleCop allow two public classes in one file?

Reply:
Got it

---
In reply to: [210467560](https://github.com/dotnet/docfx/pull/3205#discussion_r210467560) [](ancestors = 210467560,210467136)

Reply:
In V2, we have one file for JSON schema definitions, but for each JSON schema itself we have separate files, right?

---
In reply to: [210468771](https://github.com/dotnet/docfx/pull/3205#discussion_r210468771) [](ancestors = 210468771,210467560,210467136)

Reply:
yep

---
In reply to: [210468986](https://github.com/dotnet/docfx/pull/3205#discussion_r210468986) [](ancestors = 210468986,210468771,210467560,210467136)